app/assets/javascripts/filters/scoreDisplay.js

```diff
@@ -8,7 +8,7 @@ angular.module('QuepidApp')
         if (score === '?') {
           return '?';
         } else if ( angular.isNumber(score) ) {
-          return $filter('number')(score, 0);
+          return $filter('number')(score, 2);
```

Comment:
Can you walk me through this change?  I thought we dealt with this change of moving from 0 to 2 in a previous commit that we shipped?

Reply:
I don't think we did. At one point we thought about storing this information in the db's scorer table, but were unable to consistently retrieve it after the initial app load.